bindings/ruby/Rakefile

```diff
@@ -93,27 +93,14 @@ task :version do
   print GEMSPEC.version
 end
 
-desc "Resolve symlinks in core directory for gem packaging" # Read more in `.release-ruby.yml`
-task :resolve_symlinks do
-  puts "Changing symlinks in core directory to actual files for packaging..."
-  Dir.chdir("core") do # a symbolic link to OpenDAL core for packaging
-    Dir.glob("**/*", File::FNM_DOTMATCH).each do |file|
-      next unless File.symlink?(file)
-
-      link_target = File.readlink(file)
-      resolved_path = File.expand_path(link_target, File.dirname(file))
-
-      if File.exist?(resolved_path)
-        puts "Resolving symlink: #{file} -> #{link_target}"
-        File.delete(file)
-        FileUtils.cp(resolved_path, file)
-      end
-    end
-  end
+desc "Copy the core directory for gem packaging" # Read more in `./.github/workflows/release-ruby.yml`
+task :copy_core do
+  puts "Copy the core directory for packaging..."
+  FileUtils.cp_r('../../core', '.')
```

Comment:
Perhaps add to `.gitigonre` and delete every time before copy. But let's first verify a happy path can work.